code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1569,6 +1569,73 @@
 	reagents.add_reagent(NUTRIMENT, 8)
 	bitesize = 2
 
+/obj/item/weapon/reagent_containers/food/snacks/blobburger
+	name = "bloburger"
+	desc = "Careful, has a tendency to spill sauce in every direction when squeezed too hard."
```

Comment:
Could use a chance for the crumbs to be dribbles instead, with that description.